openedx/core/djangoapps/oauth_dispatch/views.py

```diff
@@ -102,15 +104,12 @@ class AccessTokenView(RatelimitMixin, _DispatchingView):
     def dispatch(self, request, *args, **kwargs):
         response = super(AccessTokenView, self).dispatch(request, *args, **kwargs)
 
-        if response.status_code == 200:
-            monitoring_utils.set_custom_metric('oauth_grant_type', request.POST.get('grant_type', ''))
-
-            if request.POST.get('token_type', '').lower() == 'jwt':
-                response.content = self._build_jwt_response_from_access_token_response(request, response)
+        token_type = request.POST.get('token_type', self.NO_TOKEN_TYPE_SUPPLIED_VALUE).lower()
```

Comment:
Nit: is it necessary to make this a constant if it's only used here?  Doesn't it keep the value too far away from the relevant code?

Reply:
Ah. right.  +1

Reply:
Actually - due to [weirdness in the test file with conditional imports](https://github.com/edx/edx-platform/blob/6110c77476da1b4137eda18567133a34e4edd3ce/openedx/core/djangoapps/oauth_dispatch/tests/test_views.py#L23-L37) - these tests were failing - so I punted and did away with the constant.  Is that ok with you?